File: Api/Dtos/PersonDto.cs
using Api.Constants;

namespace Api.Dtos;

public class PersonDto : BaseDto
{
    public string Name { get; set; }
    public string Surname { get; set; }
    public Gender Gender { get; set; }
    public string Email { get; set; }
    public string MobileNumber { get; set; }
    public Guid CountryId { get; set; }
    public string? Country { get; set; }
    public Guid CityId { get; set; }
    public string? City { get; set; }
    public string ProfilePicture { get; set; }
}

File: People.Web/src/dtos/PersonDto.ts
import { Gender } from "../constants/Gender";
import { BaseDto } from "./bases/BaseDto";

export interface PersonDto extends BaseDto {
  name: string;
  surname: string;
  gender: Gender;
  email: string;
  mobileNumber: string;
  countryId: string;
  country?: string;
  cityId: string;
  city?: string;
  hasProfilePicture: boolean;
}
